AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -768,7 +768,7 @@ byjus.com###top-banner-register
 byjus.com#?#.sidebar > div.bgc-white:has(> div.relatedlinks-widget > div.widget-content > a[onclick*="sidebar register click"])
 m.naver.com##.id_cui_commemoration_banner
 minhaoperadora.com.br##.minha-conteudo
-alibaba.com##a[campaignid][style^="display: block; width: 100%; position: relative; min-height: 120px;"]
+alibaba.com#?#[campaignid]:has(> div > div[class$="verticalCarousel"])
```

Comment:
better `alibaba.com##div[campaignid][style*="width: 100%; box-sizing: border-box;"][style*="position: relative; height:"]`
Will look how they chenge banner next time.

Reply:
Will it be long-lasting? Site changes as what the reporter said.

Reply:
So let's check)